### PR TITLE
Pandaize README.md, Fix Headline in FAQ.md

### PR DESCRIPTION
**Pull request by jomeier**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Frequently Asked Questions
+==========================
```
Comment (vrutkovs) [inline FAQ.md:2]: Eh, I had to remove that, so that VSCode would generate a nice Table of Contents. Not sure which plugin does that though

**Comment by jomeier** [comment]
It's working on my fork. Magic Josef hands, you know?

File: FAQ.md (hunk 1 above)
Comment (jomeier) [reply FAQ.md:2]: It's working. Maaagic :-) !

**Comment by vrutkovs** [comment]
>Magic Josef hands, you know?

Oh well, okay :)

**Pull request merged**